server/graphql/v2/object/Amount.js

```diff
@@ -33,5 +35,10 @@ export const Amount = new GraphQLObjectType({
         }
       },
     },
+    fxRate: {
```

Comment:
I find `fxRate` a bit jargon for a public API. It might be ok but we could replace by something like `exchangeRate`.

Reply:
Would it make sense for this field to be a list? In case we want to feature multiple rates or multiple currencies?

`exchangeRates: GraphQLList(CurrencyFxRate),`

Reply:
> I find `fxRate` a bit jargon for a public API. It might be ok but we could replace by something like `exchangeRate`.

I agree that "fxRate" sounds a bit jargon. Today we use it in `transaction.hostCurrencyFxRate`, we should ideally make the fields consistent. If that's ok for you I'll update this one to `exchangeRate` and will open an issue about renaming `hostCurrencyFxRate` to `hostCurrencyExchangeRate`



> Would it make sense for this field to be a list? In case we want to feature multiple rates or multiple currencies?
> 
> `exchangeRates: GraphQLList(CurrencyFxRate),`

This field is supposed to showcase the `fxRate` that was used to generate the amount. I don't think exposing other rates here would make sense.